cmd/wit-bindgen-go/cmd/generate/generate.go

```diff
@@ -60,6 +61,11 @@ var Command = &cli.Command{
 			Name:  "dry-run",
 			Usage: "do not write files; print to stdout",
 		},
+		&cli.BoolFlag{
```

Comment:
This should be on the root command, so any subcommand can have verbose logging.

Reply:
Good point, will move!